users/users-model.js

```diff
@@ -0,0 +1,28 @@
+const db = require('../data/dbConfig');
+
+module.exports = {
+    add,
+    find,
+    findBy,
+    findById,
+};
+
+function find() {
+    return db('users').select('id', 'username', 'password', 'department');
+}
+
+function findBy(filter) {
+    return db('users').where(filter);
+}
+
+async function add(user) {
+    const [id] = await db('users').insert(user);
```

Comment:
There is nothing wrong with what you are doing here when working with sqlite3, but destructing the id from an array, `[id]`, will throw an error with PostgreSQL